pkg/container-disk/container-disk.go

```diff
@@ -120,7 +120,7 @@ func GenerateContainers(vmi *v1.VirtualMachineInstance, podVolumeName string, po
 				// k8s minimum memory reservation is linuxMinMemory = 4194304
 				resources.Limits[kubev1.ResourceMemory] = resource.MustParse("64M")
 			}
-			containers = append(containers, kubev1.Container{
+			container := kubev1.Container{
```

Comment:
why is this necessary

Reply:
This is due to the need to have the IMAGE_PATH present only when the custom-path is added to VMI. Not in other case.

Reply:
ah I see. I think I glazed over that part. thanks